pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.ruff]
+line-length = 120
+# Enable Pyflakes `E` and `F` codes by default.
+select = [
+    "E", "W",  # see: https://pypi.org/project/pycodestyle
+    "F",  # see: https://pypi.org/project/pyflakes
+    "I",  # see: isort
+]
+ignore = [
+    "E501",
+    "E741",
+]
+ignore-init-module-imports = true
+unfixable = ["F401"]
```

Comment:
IMO there is no consistent need in this specific case (if a rule can be auto-fixable).

`F401` is a enabled rule and is will always raise a error in all case, it's "consistent" already.

Some developer (for example, myself) are using [ruff's IDE plugin](https://beta.ruff.rs/docs/editor-integrations/), which is able to fix ruff error on-save.

So make this rule auto-fixable is not productive and annoying. But it's productive in the pre-commit hooks

It's just like the autoflake8(`--remove-all-unused-imports`) pre-commit hook, we only want it to be auto-fixable in the pre-commit hooks, not when you are editing codes.